ext/oci8/oci8.stub.php

```diff
@@ -0,0 +1,814 @@
+
+/**
+ * @param resource $connection_resource
+ */
+function oci_set_client_info($connection_resource, string $client_information): bool {}
+
+/**
+ * @param resource $connection_resource
+ */
+function oci_set_db_operation($connection_resource, string $action): bool {}
+
+/**
+ * @param resource $connection_resource
+ */
+function oci_set_call_timeout($connection_resource, int $call_timeout): bool {}
+
+/**
+ * @param resource|string $connection_resource_or_connection_string
+ * @return resource|bool
+ */
+function oci_password_change($connection_resource_or_connection_string, string $username, string $old_password, string $new_password) {}
+
+/**
+ * @param resource|string $connection_resource_or_connection_string
+ * @return resource|bool
+ * @alias oci_password_change
+ * @deprecated
+ */
+function ocipasswordchange($connection_resource_or_connection_string, string $username, string $old_password, string $new_password) {}
+
+/**
+ * @param resource $connection_resource
+ * @return resource|false @todo check return false
+ */
+function oci_new_cursor($connection_resource) {}
+
+/**
+ * @param resource $connection_resource
+ * @return resource|false @todo check return false
+ * @alias oci_new_cursor
+ * @deprecated
+ */
+function ocinewcursor($connection_resource) {}
+
+/**
+ * @param resource $statement_resource
+ */
+function oci_result($statement_resource, mixed $column_number_or_name): mixed {}
+
+/**
+ * @param resource $statement_resource
+ * @alias oci_result
+ * @deprecated
+ */
+function ociresult($statement_resource, mixed $column_number_or_name): mixed {}
+
+function oci_client_version(): string {}
+
+/**
+ * @param resource $connection_resource
+ */
+function oci_server_version($connection_resource): string|false {}
+
+/**
+ * @param resource $connection_resource
+ * @alias oci_server_version
+ * @deprecated
+ */
+function ociserverversion($connection_resource): string|false {}
+
+/**
+ * @param resource $statement_resource
+ */
+function oci_statement_type($statement_resource): string|false {}
+
+/**
+ * @param resource $statement_resource
+ * @alias oci_statement_type
+ * @deprecated
+ */
+function ocistatementtype($statement_resource): string|false {}
+
+/**
+ * @param resource $statement_resource
+ */
+function oci_num_rows($statement_resource): int|false {}
+
+/**
+ * @param resource $statement_resource
+ * @alias oci_num_rows
+ * @deprecated
+ */
+function ocirowcount($statement_resource): int|false {}
+
+function oci_free_collection(OCI_Collection $collection): bool {}
+
+/**
+ * @alias oci_free_collection
+ * @deprecated
+ */
+function ocifreecollection(OCI_Collection $collection): bool {}
+
+function oci_collection_append(OCI_Collection $collection, string $value): bool {}
+
+/**
+ * @alias oci_collection_append
+ * @deprecated
+ */
+function ocicollappend(OCI_Collection $collection, string $value): bool {}
+
+function oci_collection_element_get(OCI_Collection $collection, int $index): mixed {}
```

Comment:
```suggestion
function oci_collection_element_get(OCI_Collection $collection, int $index):string|double|null {}
```
It seems that only these three return types are supported (`php_oci_collection_element_get()` says so) :)